src/Altinn.Apps/AppTemplates/AspNet/App.IntegrationTests/Mocks/Services/RegisterMockSI.cs

```diff
@@ -56,29 +56,44 @@ public async Task<Party> GetParty(int partyId)
             return null;
         }
 
-        public Task<Party> LookupParty(PartyLookup partyLookup)
+        public async Task<Party> LookupParty(PartyLookup partyLookup)
         {
-            if (!string.IsNullOrEmpty(partyLookup.Ssn))
+            string[] partyFiles = Directory.GetFiles(GetPartyFolder());
+
+            foreach (string partyFile in partyFiles)
             {
-                // TODO: fetch from disk
-                Party party = new Party
+                if (File.Exists(partyFile))
                 {
-                    PartyId = 1000,
-                    Name = "Test Lookup",
-                    SSN = partyLookup.Ssn,
-                    PartyTypeName = PartyType.Person,
-                };
+                    string content = System.IO.File.ReadAllText(partyFile);
+                    Party party = JsonConvert.DeserializeObject<Party>(content);
+
+                    if (party.OrgNumber != null && partyLookup.OrgNo != null && party.OrgNumber.Equals(partyLookup.OrgNo))
+                    {
+                        party.Organization = await _erService.GetOrganization(party.OrgNumber);
+                        return party;
```

Comment:
aren't we still missing some data on the party object that isn't stored in party.organization with this implementation?

Reply:
We might but is not that only testdata issue? Meaning those need to be filled in json